lib/errors.js

```diff
@@ -288,6 +288,11 @@ const codes = {
     "Cannot set forceCloseConnections to 'idle' as your HTTP server does not support closeIdleConnections method"
   ),
 
+  FST_ERR_DUPLICATE_SERVER: createError(
+    'FST_ERR_DUPLICATE_SERVER',
+    'Initial server passed to serverFactory will create a duplicate secondary server'
```

Comment:
Have into account that the change proposed was to throw if the server instance returned by `serverFactory` is already listening. This might need to be reworded to indicate so